### PR TITLE
Feature/button

### PR DESCRIPTION
**Pull request by GCobo**

### CODE REVIEW

**Comment by GCobo** [comment]
image

**Comment by GCobo** [comment]
image

**Comment by pedrojpj** [comment]
You are missing unit tests of the components

File: src/components/button/Button.test.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { JoinbleButton } from './Button'
```
Comment (pedrojpj) [inline src/components/button/Button.test.tsx:1]: Change for button

**Review by pedrojpj — APPROVED** [review]

**Pull request merged**